runtime/centrifuge/src/lib.rs

```diff
@@ -418,6 +430,92 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub LiquidityPoolsPalletIndex: PalletIndex = <LiquidityPools as PalletInfoAccess>::index() as u8;
+}
+
+impl pallet_liquidity_pools::Config for Runtime {
+	type AccountConverter = AccountConverter<Runtime>;
+	type AdminOrigin = EnsureRootOr<TwoThirdOfCouncil>;
+	type AssetRegistry = OrmlAssetRegistry;
+	type Balance = Balance;
+	type CurrencyId = CurrencyId;
+	type ForeignInvestment = Investments;
+	type GeneralCurrencyPrefix = cfg_primitives::liquidity_pools::GeneralCurrencyPrefix;
+	type OutboundQueue = WrapperOutboundQueue;
+	type Permission = Permissions;
+	type PoolId = PoolId;
+	type PoolInspect = PoolSystem;
+	type Rate = Rate;
```

Comment:
Very important here to change `Rate` by `Quantity` in case https://github.com/centrifuge/centrifuge-chain/pull/1520 is finally merged:

Reply:
OK, I'm not sure we wanna have that part of this release so let's keep this open until we decide that and then I'll adjust as needed.

Thank you for the catch @lemunozm! <3

cc @mustermeiszer